src/components/Bar.jsx

```diff
@@ -186,6 +186,7 @@ class Bar extends Component {
     } = this.props
     return (
       <div className="coz-bar-wrapper">
+        <div id="CozyBarSupportModalPlace" />
```

Comment:
Since we will have to use it for all cozy-bar modals, maybe we can name it just `CozyBarModalDomPlace` ?

Reply:
Why not, if we never have more than one modal at the same time ( that should be OK, but ... )

Reply:
The bar is not designed to displayed multiple modals but it could have different modals ;)

Reply:
Great you found the solution.

I think that the `id` should not be CamelCased but use hypen, like our CSS classes and other `id` attributes in the Cozy-Bar codebase ([example](https://github.com/cozy/cozy-bar/blob/master/src/components/Apps/index.jsx#L67)).